static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -128,6 +126,10 @@ const addFieldAsColumn = (
       field: newField,
     },
   });
+  trackAdvancedAnalyticsEvent('discover_v2.quick_context_add_column', {
+    organization: organization || null,
+    column: fieldName,
+  });
```

Comment:
I would move this above the `browserHistory.push` just in case the push causes a race condition and messes with the analytics request

Reply:
Done